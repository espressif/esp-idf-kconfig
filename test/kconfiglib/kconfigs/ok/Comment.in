# Test comment keyword
mainmenu "Test comments"

    comment "always visible comment"

    config COMMENTS_SHOWN
        bool
        default y

    comment "Visible conditional comment"
        depends on COMMENTS_SHOWN

    comment "Invisible conditional comment"

    config BELOW_COMMENT
        bool
        default y
