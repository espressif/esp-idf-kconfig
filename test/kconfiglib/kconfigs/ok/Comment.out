#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#

#
# always visible comment
#
CONFIG_COMMENTS_SHOWN=y

#
# Visible conditional comment
#

#
# Invisible conditional comment
#
CONFIG_BELOW_COMMENT=y
